reflex/event.py

```diff
@@ -25,6 +25,8 @@ class Event(Base):
     # The event payload.
     payload: Dict[str, Any] = {}
 
+    type_: Optional[str] = None
```

Comment:
Have this here as a means to identify what event is an upload event. Is this a good enough approach? Are there better alternatives? The name seems a bit misleading though. What's a better one (assuming this is the right approach, of course)